File: doc/_i18n/en/tutorials/introduction/ef-controller/create_ef_task.py
# Import the mc_tasks module
import mc_tasks
# In the constructor, create the task and add it to the problem
self.efTask = mc_tasks.EndEffectorTask("l_wrist", self.robots(), 0, 10.0, 1000.0)
self.qpsolver.addTask(self.efTask)
# In the reset callback, reset the task to the current EF position
self.efTask.reset()

File: doc/_i18n/en/tutorials/introduction/ef-controller/create_ef_task.cpp
// Include the EF task header (header)
#include <mc_tasks/EndEffectorTask.h>
// In the class private members (header)
std::shared_ptr<mc_tasks::EndEffectorTask> efTask;
// In the constructor, create the task and add it to the problem
efTask = std::make_shared<mc_tasks::EndEffectorTask>("l_wrist", robots(), 0, 5.0, 500.0);
solver().addTask(efTask);
// In the reset function, reset the task to the current EF position
efTask->reset();
